src/main/java/mate/zorii/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -33,4 +34,19 @@ public BookDto findById(Long id) {
         return bookMapper.toDto(bookRepository.findById(id).orElseThrow(
```

Comment:
If you would like to calling method in line, let's make it more readable 
```suggestion
        return bookRepository.findById(id)
                .map(bookMapper::toDto)
                .orElseThrow(() -> new EntityNotFoundException("No book found by id " + id)));
```